CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+## Contributors guidelines
+ 
+### Getting started
+To start contributing to the Onfido Web SDK, clone this repository and run the following commands:
+ 
+- `npm install` to install the project dependencies
+- `npm run dev` to generate the build and run the dev server
+ 
+The SDK will be running on https://localhost:8080.
+ 
+Note: The SDK runs on `https`, which is needed to support the camera functionality.
+ 
+You can also run it using `docker` and `docker-compose` by running the following command: `docker-compose -f docker-compose.dev.yml`.
```

Comment:
I think, from my admittedly little experience of `docker-compose` so far, this should be `docker-compose -f docker-compose.dev.yml up`